source/Playnite/Emulation/Platforms.yaml

```diff
@@ -376,6 +376,16 @@
   IgdbId: 6
   Emulators: [pcem, scummvm]
 
+- Name: Linux

```

Comment:
This should be imho `PC (Linux)` for consistency and mac should be `Macintosh`.

Reply:
I'm fine with that.  I just used the naming that IGDB used, as I thought that's what you were going for.  It uses "PC (Windows)" and "PC (DOS)", but "Linux" and "Mac" rather than "PC (Linux)" and "Macintosh".  Those are obviously all just arbitrary choices, so it's all the same to me as long as the types themselves are included.

Due to the error I caught above (re: not having submitted this off its own branch, and therefore it including other random stuff) I'll retract this PR and resubmit.  Unless you want to just insert the two types in yourself, certainly fine by me too.

Reply:
We don't use names from IGDB, so I don't see why we should use it for these two. If some match with what's on IGDB, it's just a coincidence. Most of the naming is based on what libretrodb uses.

Reply:
Sounds good to me.  Like I said, it's all just arbitrary choices.  I had assumed IGDB since the current ones all match it exactly. (Libretro, to my knowledge, doesn't have PC/Windows/Linux/Mac platforms specified *at all*; it does have a DOS platform, but while ours and IGDB's is called "PC (DOS)" libretro's is just called "DOS".)

I should have this updated in a few minutes.

Reply:
Just to make sure: "Macintosh" and not "MacOS" and/or "OS X"?  Again, it's just a personal choice; just wanted to present the option.  (I have a Mac that runs Linux on it, so whatever software I run on it would fall under both platforms "PC (Linux)" and "Macintosh" -- or neither, perhaps? Who knows, this platform stuff is messy no matter what. Hence arbitrary choices) :)

Reply:
Renamed as requested.

Reply:
I don't know, it's arbitrary. If we were to go by the current naming, then it should be "macOS" :)